composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "yajra/laravel-datatables-oracle",
+  "name": "vladrub/laravel-datatables-oracle",
```

Comment:
Thanks but changing the package name will break the package. Please revert this one.